spec/models/rental_unit_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe RentalUnit, type: :model do
+  it { should belong_to :tenant }
+  it { should belong_to :landlord }
+  # it { should belong_to :electric_utility }
+  # it { should belong_to :gas_utility }
+  # it { should belong_to :neighborhood }
+  # it { should have_many :bills }
+
+  it { should validate_presence_of :address }
+
+  it { should validate_presence_of :number_of_bedrooms }
+
+  it { should validate_numericality_of(:number_of_bedrooms).only_integer }
+  it { should validate_numericality_of(:number_of_bedrooms).
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.